contrib/docker-compose/standalone/docker-compose.yml

```diff
@@ -82,12 +84,10 @@ services:
     image: haproxy:2.6-alpine
     user: root
     restart: always
-    environment:
-    - RELOAD_INTERVAL=0
     volumes:
-    - ./haproxy-entrypoint.sh:/usr/local/bin/haproxy-entrypoint.sh
-    - ./haproxy/etc/haproxy-internal.cfg:${HAP_CONF_DIR}/haproxy.cfg
-    entrypoint: "/usr/local/bin/haproxy-entrypoint.sh -f ${HAP_CONF_DIR}/haproxy.cfg"
+    - ./haproxy-internal-entrypoint.sh:/usr/local/bin/haproxy-entrypoint.sh
```

Comment:
I don't see the `internal` file in the commit

Reply:
good catch, that line should have been deleted (I decided to inline the internal custom entrypoint because it was so small)